test/sending.rho

```diff
@@ -0,0 +1,65 @@
+/**
+ * sending - send REV from registered wallet to party identified by pk.
+ */
+new send, makePayeeWallet,
+    rl(`rho:registry:lookup`),
+    ri(`rho:registry:insertArbitrary`)
+in {
+  exports!(*send) |
```

Comment:
When I wrote the beginnings of rhopm, I used a comment for imports and exports. Sending to an export channel has a nice rholang-y feel to it, but the disadvantage is that this file is not valid rholang as it stands because of top-level free-variable `exports`.

Interestingly, when I tried to eval this rholang file to get the exact error message, I got `Free variable trace is used twice as a binder (at 18:9 and 29:11) in name context.` instead. What am I missing?

Reply:
Missing? I don't see that you're missing anything. `exports` and `trace` are both free top-level variables.

The `used twice as a binder` is not a very good diagnostic, but that's about par for the course, in my experience.

Reply:
And yes, the "not valid rholang" disadvantage is a big one. Hmm...